platform/platform-app/src/main/resources/features.xml

```diff
@@ -776,6 +776,12 @@
         <bundle>mvn:ddf.platform.country/platform-country-converter-local/${project.version}</bundle>
     </feature>
 
+    <feature name="platform-usng4j" install="manual" version="${project.version}"
```

Comment:
@dcruver @harrison-tarr  Is it necessary to have platform-usng4j automatically install when platform-app gets installed or would we rather have this feature install when another feature that depends on platform-usng4j gets installed?